contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -329,6 +329,17 @@ public void testOpenProcess() {
         assertEquals(WinError.ERROR_ACCESS_DENIED, Kernel32.INSTANCE.GetLastError());
     }
 
+    public void testQueryFullProcessImageName() {
+        HANDLE h = Kernel32.INSTANCE.OpenProcess(0, false, Kernel32.INSTANCE.GetCurrentProcessId());
```

Comment:
It's just a test, but you should `CloseHandle` on the returned value here, MSDN says you need to.